styles.css

```diff
@@ -0,0 +1,140 @@
+body {
+    margin: 0;
+    padding: 0;
+    text-align: center;
+    font-family: 'Poppins', Arial, Helvetica, sans-serif;
+    color: #271D19;
+    background: #F6F1EF;
+}
+
+a {
+    text-decoration: none;
+    color: white;
+  }
+  a:hover {
+    color: #94563D;;
+  }
+  
+  .main-nav {
+    width: 45%;
+    background: #EC784A;
```

Comment:
The white links on the #EC784A background [only have a 2.85:1 contrast ratio](https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=EC784A).